src/main/java/net/cytonic/cytosis/data/enums/CytosisPreferences.java

```diff
@@ -28,7 +27,7 @@ private CytosisPreferences() {
     /**
      * A preference to store the players chat channel, type of ChatChannel
      */
-    public static final NamespacedPreference<ChatChannel> CHAT_CHANNEL = new NamespacedPreference<>(NamespaceID.from("cytosis", "chat_channel"), ChatChannel.ALL);
+    public static final NamespacedPreference<String> CHAT_CHANNEL = new NamespacedPreference<>(NamespaceID.from("cytosis", "chat_channel"), "ALL");
```

Comment:
No. This is monkey patch. I'm literally working on making it support enums RIGHT NOW.